### DISCUSSION TITLE
Cadmium

### DISCUSSION THREAD

**Discussion by LoganMD** [Maccraft123/Cadmium#17] [General]
Mmmm cadmium![RDT_20210528_1440377823171609410780404.jpg](https://user-images.githubusercontent.com/25466853/120066865-5fe77e00-c02d-11eb-97bb-8f4d5e37c5cd.jpg)

**Comment by Maccraft123** [comment]
what

**Reply by LoganMD** [reply]
cadmium

**Comment by Maccraft123** [comment]
cadmium

**Comment by chinh4** [comment]
cadmium
